### ISSUE TITLE
add motorway and trunk links to roads_gen1

### ISSUE THREAD

**Issue by tbonfort** [MapServer/basemaps#12]

**Comment by sgrossberndt** [comment]
The imposm3 mapping contains those: https://github.com/mapserver/basemaps/blob/master/imposm3-mapping.json#L20 - mark as solved? Or add it to https://github.com/mapserver/basemaps/blob/master/imposm-mapping.py#L368 too?

**Comment by yjacolin** [comment]
yes, we should add it alos in imposm-mapping.py or only support for imposm3. Any though? I am for support both.

**Comment by sgrossberndt** [comment]
That would mean they are currently only displayed in zoomlevel 10: https://github.com/mapserver/basemaps/blob/master/generate_style.py#L279 (which is missing the `order by z_order asc` by the way - is that on purpose?) as the zoomlevels 8 and 9 use additional restrictions in the data query.
